#异常

# a = "123"
# b = [1,2,3]
# try:
#     print(b[4])
# except:
#     print("出错了")
# finally:
#     print("没有错")

